symfony/plugins/orangehrmAdminPlugin/Api/LicenseAPI.php

```diff
@@ -172,7 +172,11 @@ public function saveLicense(): License
     public function getValidationRuleForCreate(): ParamRuleCollection
     {
         return new ParamRuleCollection(
-            new ParamRule(self::PARAMETER_NAME),
+            new ParamRule(
+                self::PARAMETER_NAME,
+                new Rule(Rules::STRING_TYPE),
+                new Rule(Rules::LENGTH, [null, 120]),
```

Comment:
wrong validation, `new Rule(Rules::LENGTH, [null, 100]),` check the schema

Reply:
@RajithaKumara any chance define this char length as class const?

Reply:
```php
    public const PARAM_RULE_NAME_MAX_LENGTH = 100;
    // ...

            new ParamRule(
                self::PARAMETER_NAME,
                new Rule(Rules::STRING_TYPE),
                new Rule(Rules::LENGTH, [null, self::PARAM_RULE_NAME_MAX_LENGTH]),
            ),

```

how about this?